src/DynamoCore/Library/DocumentationServices.cs

```diff
@@ -76,14 +76,33 @@ public static XmlReader GetForAssembly(string assemblyPath, IPathManager pathMan
             if (File.Exists(documentationPath))
                 return true;
 
+            //try with uppercase XML for linux
+            documentationPath = PathHelper.replaceExtension(documentationPath, ".XML");
```

Comment:
yes okay, I'll look into that, the issue is that we want to pick them in the order this method specifies - and then I will add a debug condition log that prints the loaded file location for documentation.